dev-lang/nasm/patches/nasm-2.14.02.patchset

```diff
@@ -0,0 +1,270 @@
+From 4f2b8256a735d00ef50d76abeb56b772d8f3b2ed Mon Sep 17 00:00:00 2001
```

Comment:
Please switch to [this] patch instead, which fix the actual root of the problem.

[this]: http://ix.io/289y

Reply:
Thanks. I'll update this later.